package.json

```diff
@@ -31,14 +31,17 @@
     "@babel/register": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
     "babel-loader": "7.1.1",
+    "core-js": "^3.0.1",
```

Comment:
This is necessary to support `yarn link` usage since `node` ends up looking for `core-js` in math-input's `node_modules` folder when loading math-input when it's symlinked in.